sql/query/expr/literal.go

```diff
@@ -93,6 +99,22 @@ func (l LiteralExprList) IsEqual(other Expr) bool {
 	return true
 }
 
+// String implements the fmt.Stringer interface.
```

Comment:
Some `String` methods miss this comment. We should add it to each exported String methods.